Makefile

```diff
@@ -139,6 +139,10 @@ test: latest-csb lint run-integration-tests ## run the tests
 run-integration-tests: latest-csb ## run integration tests for this brokerpak
 	cd ./integration-tests && go run github.com/onsi/ginkgo/v2/ginkgo -r .
 
+.PHONY: run-terraform-tests
```

Comment:
Are you adding this step to run in Github Action and CI as well?

Reply:
We have to look more careful for github actions as it requires IaaS credentials. None of the tests we run so far need that.